src/DynamicModel.hpp

```diff
@@ -2,6 +2,7 @@
 #define _DYNAMIC_MODEL_H_
 
 #include "DataTypes.hpp"
+#include <stdexcept>
```

Comment:
when building I am getting an error that 'runtime_error' is not a member of 'std'